pyroomacoustics/signal.py

```diff
@@ -0,0 +1,73 @@
+# Signal object for simulating source signals. 
+# Copyright (C) 2019  Robin Scheibler, Ivan Dokmanic, Sidney Barthe
```

Comment:
This should probably be your name here :)

Reply:
I was not sure how you handle these - whoever edits the file adds their name there?

Reply:
I think so. If you write the code, you own the copyright to that code. When multiple people edit or modify the file, I'm not comletely sure, but my understanding is that you'll list all the people who contributed to the file.